ci-operator/step-registry/ipi/conf/aws/proxy/ipi-conf-aws-proxy-commands.sh

```diff
@@ -0,0 +1,398 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+function generate_proxy_ignition() {
+cat > /tmp/proxy.ign << EOF
+{
+  "ignition": {
+    "config": {},
+    "security": {
+      "tls": {}
+    },
+    "timeouts": {},
+    "version": "2.2.0"
+  },
+  "passwd": {
+    "users": [
+      {
+        "name": "core",
+        "sshAuthorizedKeys": [
+          "${ssh_pub_key}"
+        ]
+      }
+    ]
+  },
+  "storage": {
+    "files": [
+      {
+        "filesystem": "root",
+        "path": "/tmp/squid/passwords",
+        "user": {
+          "name": "root"
+        },
+        "contents": {
+          "source": "data:text/plain;base64,${HTPASSWD_CONTENTS}"
+        },
+        "mode": 420
+      },
+      {
+        "filesystem": "root",
+        "path": "/tmp/squid/squid.conf",
+        "user": {
+          "name": "root"
+        },
+        "contents": {
+          "source": "data:text/plain;base64,${SQUID_CONFIG}"
+        },
+        "mode": 420
+      },
+      {
+        "filesystem": "root",
+        "path": "/tmp/squid.sh",
+        "user": {
+          "name": "root"
+        },
+        "contents": {
+          "source": "data:text/plain;base64,${SQUID_SH}"
+        },
+        "mode": 420
+      },
+      {
+        "filesystem": "root",
+        "path": "/tmp/squid/proxy.sh",
+        "user": {
+          "name": "root"
+        },
+        "contents": {
+          "source": "data:text/plain;base64,${PROXY_SH}"
+        },
+        "mode": 420
+      }
+    ]
+  },
+  "systemd": {
+    "units": [
+      {
+        "contents": "[Service]\n\nExecStart=bash /tmp/squid.sh\n\n[Install]\nWantedBy=multi-user.target\n",
+        "enabled": true,
+        "name": "squid.service"
+      },
+      {
+        "dropins": [
+          {
+            "contents": "[Service]\nExecStart=\nExecStart=/usr/lib/systemd/systemd-journal-gatewayd \\\n  --key=/opt/openshift/tls/journal-gatewayd.key \\\n  --cert=/opt/openshift/tls/journal-gatewayd.crt \\\n  --trust=/opt/openshift/tls/root-ca.crt\n",
+            "name": "certs.conf"
+          }
+        ],
+        "name": "systemd-journal-gatewayd.service"
+      },
+      {
+        "enabled": true,
+        "name": "systemd-journal-gatewayd.socket"
```

Comment:
We probably don't need to stream out journal entries, if the plan is to SCP down logs post-test.